contrib/ros/README.md

```diff
@@ -1,199 +1,38 @@
-# rllab.contrib.ros Instructions
```

Comment:
Realistically, the only thing 99% of the people visiting this page want to see is the install instructions. They don't care who wrote the software, why they wrote it, or what the architecture is. The purpose of writing software and sharing it is so that everyone else can enjoy not caring about these things. In that way, good open source software requires a lot of humility.

The purpose of a README is to give the shortest possible introduction to a piece of software, and get me running the software as fast as possible. You can move the install instructions into their own page if you feel like it, but a giant link to the install instructions should be within the first 10 lines of this page.

I don't think the Gazebo section provides any relevant information to users.

The architecture diagram may be helpful, I don't know because the link is broken for me.

Sawyer is not so much an example as the only functionality we have right now. It's important to be honest about that.

People don't really care what your future directions are, they want to know what you can do for them today. Preferably fast.

It's not helpful to post a reference to the repository your work is a part of. That's circular. The only reason the rllab README includes a reference is so that people know how to cite the repository in their research papers.

This rewrite of the documentation distracts a lot from the PR. Please make it is own PR.

The change is "Refactor rllab ROS". Therefore, the PR should only contain updates to the documentation which are affected by the refactor.